libraries/aws_ssm_association.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsSsmAssociation < AwsResourceBase
+  name 'aws_ssm_association'
+  desc 'Verifies settings for a SSM Association'
+
+  example "
+    describe aws_ssm_association(association_id: 'association-id') do
+      it { should exist }
+    end
+  "
+
+  def initialize(opts = {})
+    opts = { association_id: opts } if opts.is_a?(String)
+    super(opts)
+    validate_parameters(required: [:association_id])
```

Comment:
```Describes the association for the specified target or instance. If you created the association by using the Targets parameter, then you must retrieve the association by using the association ID. If you created the association by specifying an instance ID and a Systems Manager document, then you retrieve the association by specifying the document name and the instance ID.```
https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/SSM/Client.html#describe_association-instance_method
Looks like `describe_association` method supports association lookup with `instance id + document name` .
I think we should support this as well.